kernel/kernel-biosdk-provider/src/main/java/io/mosip/kernel/biosdk/provider/util/BIRConverter.java

```diff
@@ -110,8 +110,8 @@ public static BIR convertToBIR(io.mosip.kernel.biometrics.entities.BIR bir) {
                         .withMinor(bir.getVersion().getMinor())
                         .withMajor(bir.getVersion().getMajor()).build())
                 .withCbeffversion(new BIRVersion.BIRVersionBuilder()
-                        .withMinor(bir.getCbeffversion().getMinor())
-                        .withMajor(bir.getCbeffversion().getMajor()).build())
+                        .withMinor(bir.getCbeffVersion().getMinor())
```

Comment:
There should no such method name change in the API. Otherwise it will break any existing implementations done by BIO SDK providing vendors.

Reply:
reverting for now but we have to add a cleanup task as it should be in camel case.